doc/attr.xml

```diff
@@ -8,6 +8,42 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="ArticulationPoints">
+<ManSection>
+  <Attr Name="ArticulationPoints" Arg="digraph"/>
+  <Returns>A list of vertices.</Returns>
+  <Description>
+
+    A connected digraph is <E>biconnected</E> if it is still connected (in the
+    sense of <Ref Prop="IsConnectedDigraph"/>) if any vertex is removed. 
```

Comment:
Replace `if any vertex` by `when any vertex` to avoid a slightly confusing double-if.